src/openapi/transport/retry.js

```diff
@@ -44,7 +45,13 @@ function transportMethod(method) {
  * @param {object.<string,object>} [options.methods] - Http methods that should retry. For each method provide an object with `retryLimit` parameter.
  * @example
  * // Constructor with parameters
- * var transportRetry = new TransportRetry(transport, {retryTimeout:10000, methods:{'delete':{retryLimit:3}}});
+ * var transportRetry = new TransportRetry(transport, {
+ *      retryTimeout:10000,

```

Comment:
Post merge question :)
How this retryTimeout relates to ones declared in methods?
Is this default if retryTimeouts are not provied for given methods?